marshmallow_objects/models.py

```diff
@@ -238,8 +239,13 @@ def validate(cls, data, context=None, many=None, partial=None):
         return schema.validate(data, many=many, partial=partial)
 
     def __copy__(self):
+        """A magic method to implement shallow copy behavior."""
         return self.__class__.load(self.dump(), context=self.context)
 
+    def __deepcopy__(self, memo):
```

Comment:
So, you just copy/paste-d the code, why don't you want to use `self.__copy__()`?. Also I would suggest to store the id(self) in the memo:
```python
obj = self.__copy__()
memo[id(self)] = obj
return obj
```

Reply:
Ok, I see